VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -97,6 +99,8 @@ public void notifyNetworkConnect(){
         }
     }
 
+    // This flag is to enable the login and sign-up Activities to be
+    // excluded from the download progress update callback.
     protected boolean runOnTick = true;
```

Comment:
This should be renamed to be more verbose, if the purpose of it is to subscribe/unsubscribe from progress callbacks. Something like `isDownloadProgressSubscribed` (can be made a little better :wink: ).
This will also help us in fixing https://openedx.atlassian.net/browse/MA-1665.